android/src/main/java/dev/barooni/capacitor/calendar/CapacitorCalendarPlugin.kt

```diff
@@ -51,9 +51,9 @@ class CapacitorCalendarPlugin : Plugin() {
             intent.putExtra(CalendarContract.Events.TITLE, title)
             calendarId?.let { intent.putExtra(CalendarContract.Events.CALENDAR_ID, it) }
             location?.let { intent.putExtra(CalendarContract.Events.EVENT_LOCATION, it) }
-            startDate?.let { intent.putExtra(CalendarContract.Events.DTSTART, it) }
-            endDate?.let { intent.putExtra(CalendarContract.Events.DTEND, it) }
-            isAllDay?.let { intent.putExtra(CalendarContract.Events.ALL_DAY, if (it) 1 else 0) }
+            startDate?.let { intent.putExtra(CalendarContract.EXTRA_EVENT_BEGIN_TIME, it) }
+            endDate?.let { intent.putExtra(CalendarContract.EXTRA_EVENT_END_TIME, it) }
+            isAllDay?.let { intent.putExtra(CalendarContract.EXTRA_EVENT_ALL_DAY, if (it) 1 else 0) }
```

Comment:
@ebarooni this doesn't seem to work yet. I think it's expecting a boolean, not a number.